package.json

```diff
@@ -16,16 +16,37 @@
     "dbus",
     "api"
   ],
-  "main": "lib/nm.js",
+  "main": "dist/index.js",
   "scripts": {
+    "start": "node dist/index.js",
+    "build": "tsc lib/index.ts --outDir dist/",
     "test": "mocha",
+    "build-test": "npm run build && npm run test",
     "lint": "standard lib && standard --env mocha test",
     "lint-fix": "standard --fix lib && standard --fix --env mocha test"
   },
-  "dependencies": {},
+  "dependencies": {
+    "bluebird": "^3.5.1",
+    "dbus-native": "^0.2.5",
+    "express": "^4.16.3",
+    "lodash": "^4.17.10",
+    "typed-error": "^3.0.0",
+    "uuid": "^3.3.2"
+  },
   "devDependencies": {
+    "@types/bluebird": "^3.5.21",
+    "@types/express": "^4.16.0",
+    "@types/lodash": "^4.14.110",
+    "@types/mocha": "^5.2.4",
+    "@types/node": "^10.5.1",
+    "@types/uuid": "^3.4.3",
     "mocha": "^5.2.0",
-    "standard": "^11.0.1"
+    "standard": "^11.0.1",
+    "ts-loader": "^4.4.2",
+    "typescript": "^2.9.2",
+    "webpack": "^4.14.0",
+    "webpack-cli": "^3.0.8",
+    "webpack-node-externals": "^1.7.2"
```

Comment:
Webpack doesn't seem to be used anywhere - is this also a leftover?

Reply:
Yeah, removed and pushed update.